backend/main/shared/flow.go

```diff
@@ -356,6 +421,30 @@ func WaitForSeal(
 	return result, tx, errTx
 }
 
+func (fa *FlowAdapter) CreateNFTCollection(ctx context.Context) {
```

Comment:
where is this being called?
i just want to verify that it couldn't get run somehow in production, although it probably wouldn't work anyway.

Reply:
not being called anywhere yet, working on that part still